ui/cryptomaterial/dropdown.go

```diff
@@ -3,14 +3,16 @@ package cryptomaterial
 import (
 	"image/color"
 
+	"gioui.org/font"
 	"gioui.org/layout"
 	"gioui.org/unit"
 	"gioui.org/widget"
 	"github.com/crypto-power/cryptopower/ui/values"
 )
 
 const (
-	DropdownBasePos uint = 0
+	DropdownBasePos     = 0
+	maxDropdownItemChar = 20
```

Comment:
maxDropdownItemTextLen.

Add a comment as well. On face value, it sounds like the item text can't be more than 20 chars, but it can, just that it will be truncated and an ellipsis will be shown at the end.